pkg/resource/parameter_group/post_set_output.go

```diff
@@ -0,0 +1,90 @@
+package parameter_group
```

Comment:
super nit: Add Amazon copyright headers to custom files. (I'm not sure if this is required tbh)